.circleci/config.yml

```diff
@@ -542,12 +557,17 @@ workflows:
           filters:
             tags:
               only: /v\d+\.\d+\.\d+(-[\w]+)?/
+      - cli-extra-test:
+          name: cli-extra-test
+#          requires:  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker